src/Fantomas.CoreGlobalTool/Program.fs

```diff
@@ -29,6 +29,7 @@ type Arguments =
     | [<Unique;AltCommandLine("--keepNewlineAfter")>] KeepNewlineAfter
     | [<Unique;AltCommandLine("--maxIfThenElseShortWidth ")>] MaxIfThenElseShortWidth of int
     | [<Unique;AltCommandLine("--strictMode")>] StrictMode
+    | [<Unique;AltCommandLine("--emptyLineBeforeNestedModuleBody")>] EmptyLineBeforeNestedModuleBody
```

Comment:
This is something you couldn't have known but moving forward we don't want any new settings in the CoreGlobalTool.
Users should pass them via the `--config` option and list them in a file.
In the next major version, we will probably remove all CLI settings related to stylistic preference.